chunky/src/java/se/llbit/chunky/world/region/ImposterCubicRegion.java

```diff
@@ -433,23 +436,23 @@ public void parse() {
             }
 
             if (length < (long)sectorOffset * SECTOR_SIZE + Integer.BYTES) {
-              System.err.printf("Cube (%d, %d, %d) is outside of region file %s! Expected chunk data at offset %d but file length is %d%n",
-                cubicRegionToCube(regionX, localX), cubicRegionToCube(regionY, localY), cubicRegionToCube(regionZ, localZ), fileName, sectorOffset * SECTOR_SIZE_BYTES, length);
+              Log.warn(String.format("Cube (%d, %d, %d) is outside of region file %s! Expected chunk data at offset %d but file length is %d%n",
```

Comment:
I think there is a Log.warnf function if you want (sorry for noting that earlier)

Reply:
oh... :facepalm: